pandas/core/base.py

```diff
@@ -702,7 +702,8 @@ def shape(self):
     @property
     def ndim(self):
         """
-        Return the number of dimensions of the underlying data, by definition 1.
+        Return the number of dimensions of the underlying data,
+        by definition 1.
```

Comment:
Sorry about that @avolkov, but we'll validate soon that the first paragraph in the docstrings is a single line (it's shown in the summaries, and if it's too long the format gets wrong and things are difficult to read).

Do you mind removng the `Return the`, and start in the `Number of dimensions...` so this fits in a single line?